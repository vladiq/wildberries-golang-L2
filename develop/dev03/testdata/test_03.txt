8234
324
12
1
0
233
2
321