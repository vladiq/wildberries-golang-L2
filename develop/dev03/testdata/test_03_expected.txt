0
1
2
12
233
321
324
8234